src/cnc/models/providers/amazon/environment_collection.py

```diff
@@ -172,16 +172,14 @@ def get_secret_value(self, secret_id, version_id=None):
         return secret_string
 
     def generate_tf_assets(self, config_files_path, rendered_files_path):
-        if self.application.flavor == "lambda-lite":
-            lambda_payload_path = os.path.join(
-                rendered_files_path, "lambda_function_payload"
-            )
-            shutil.make_archive(
-                lambda_payload_path,
-                "zip",
-                root_dir=(f"{config_files_path}/lambda_function_payload"),
-            )
-            return True
+        lambda_payload_path = os.path.join(
```

Comment:
Instead of removing the condition here, we should update it to something along the lines of `if env_collection.has_serverless_services` - also since we're zipping the lambda here, are we using `assets/lambda_function_payload.zip` anywhere? if not probably best to remove it

Reply:
Fixed. PR updated.